app/gui/view/graph-editor/src/component/node/input/widget.rs

```diff
@@ -165,6 +166,11 @@ pub struct OverrideKey {
 pub trait SpanWidget: display::Object {
     /// Configuration associated with specific widget variant.
     type Config: Debug + Clone + PartialEq;
+    /// Declare whether this widget should be considered for creation despite a config override
+    /// being present. Widgets that declare `true` here should always create a child widget on
+    /// the same span-tree node, so that the override can eventually be applied to it. Only widgets
+    /// declared before the widget selected by override will take priority over it.
+    const QUERY_ON_OVERRIDE: bool = false;
```

Comment:
I don't understand the last sentence. They will take priority over what? What are the consequences of taking priority?
Also, the working of the second sentence is also confusing. I suggest to slightly rewrite it as follows:
```
Widgets that declare `true` here must create a child widget on the same span-tree node, so that the override can be applied to this child instead.
```

Reply:
I've added more docs explaining the implicit ordering and where it is defined.